app/layout_elements.py

```diff
@@ -20,36 +21,55 @@ def display_costs(
     key_suffix = key_suffix.lower().replace(" ", "_")
     st.subheader(titlestring)
 
-    # filter data:
-    df_res = df_costs.copy()
+    c1, c2 = st.columns(2)
 
-    # select filter:
-    show_which_data = st.radio(
-        "Select elements to display:",
-        ["All", "Manual select"],
-        index=0,
-        horizontal=True,
-        key=f"show_which_data_{key}_{key_suffix}",
-    )
+    # select which dataset to display:
+    if st.session_state["user_changes_df"] is not None:
+        with c2:
+            st.info("Input data has been modified. Select which data to display.")
+            select_data = st.radio(
+                "Select data to display",
+                ["with user modifications", "without user modifications", "difference"],
+                horizontal=True,
+                key=f"select_data_{key}_{key_suffix}",

```

Comment:
you get duplicate key error when clicking manual select in the left column 
```suggestion
                key=f"select_user_modificatons_data_{key}_{key_suffix}",

```